AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListItem.swift

```diff
@@ -55,6 +62,62 @@ extension ItemListItem {
                   accountName: username,
                   itemType: .totp(model: totpModel))
     }
+
+    /// Initialize an `ItemListItem` from an `AuthenticatorItemView`
+    ///
+    /// - Parameters:
+    ///   - authenticatorItemView: The `AuthenticatorItemView` used to initialize the `ItemListItem`
+    ///
+    init?(itemView: AuthenticatorBridgeItemDataView, timeProvider: TimeProvider) {
+        guard let totpKey = TOTPKeyModel(authenticatorKey: itemView.totpKey) else { return nil }
+        let totpCode = TOTPCodeModel(code: "123456", codeGenerationDate: timeProvider.presentTime, period: 30)
```

Comment:
🤔 Why is the code hardcoded here?

Reply:
This confused me at first as well. I picked that up from the other `init?` here. The code is just set to provide a non-nil value which is then replaced anyway when the ItemListProcessor first grabs the items to show.

I changed it to simply be empty (`""`) which is hopefully less confusing? I could also see defining a `static let default = "123456"` and assigning that, which would at least call this out as a default.

@KatherineInCode I don't see any issues with changing it to `""` - I just had to update a few test assumptions. Do you see any issue with changing this?

Reply:
It was because I wanted the `code` in a `TOTPCodeModel` to be non-optional, but still needed something there on initialization, before we knew what the code *was*. When I did [PR 119](https://github.com/bitwarden/authenticator-ios/pull/119) I fiddled around a bit with having it get the actual code from the `TOTPService` but that caused other problems that seemed out of scope for that PR. I'd eventually like to swing back around and take another look at it, but I figure it's "good enough" for now.

That said, I think it's fine to make it an empty string; I'd done "123456" so that I also had a code to display, I think.

Reply:
Alright makes sense but I'd love to see if we could have another way of doing this without resorting to these default values that the caller needs to add. Could a comment be added on this indicating why that is needed? I feel like if another dev sees it, they may be confused as well as to why that was introduced.

Reply:
In the last commit, I went ahead and added a constant in the `ItemListItem` for this. Then I added a doc comment to that explaining why it's needed and what it's doing. I also moved it back to "123456" since it's now more clear why it's there.  👍